### ISSUE TITLE
[Bug Report][3.1.11] Accessibility: VTextField using the hide-details prop adds an 'aria-describedby' attribute referencing a non-existent element

### ISSUE THREAD

**Issue by ryanzbartlett** [vuetifyjs/vuetify#17012]
### Environment
**Vuetify Version:** 3.1.11
**Vue Version:** 3.2.47
**Browsers:** Chrome 111.0.0.0
**OS:** Mac OS 10.15.7

### Steps to reproduce
Use the `hide-details` (or `hide-details="auto"`) prop on a VTextField component.

### Expected Behavior
The `aria-describedby` attribute should not be present, or should be empty or null.

### Actual Behavior
The `aria-describedby` attribute refers to a messages element that doesn't exist, causing accessibility check errors.

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqdUstu2zAQ/JUNL2oBS4LtAgUMp0hv/YACPUQBQktrmw1fIFduAsP/3qUku6ryAgJdtLPLmdkhb4/iu/fFoUWxEmtC47Uk/FZZgPUhl953v11hpLJc9XWHED5SvlWoG1DNdSVM3M0rATzqGtT/gHIgAbgcvspzpqL++7nHiFAHZGmQFu5lUDJvMNZBbbDZPN2DJOL/lvsBtxjQ1sruzmyQDqFGg5aA9pKgcRhtRoCPKtJZKM8vPl42v5iaT8BeNcheSCodR5u8zLCcMiRgzMCobMkNoQzZlpdw+6KPfV2OroPLlIYniEhtaivjXSA4pkDgBNvgDGR8j1karp2NBCl+uE4Dn7IfqLWDXy7o5ir7PJpYvDuxfG1iXfae2I2Yid4Qb+KL39FZfk/HtFA1NGIlVtAhCWOjqa7EnsjHVVm21j/sitqZ8oZ7ZWgtKcOxOXOzLBbFl69lw1c5xguMJt8E9ydiYMVKzEbkpLZPrwkMbeadF/N5zztguZabmIifEZascsCQ89Nr+P2FN91PZv/bYNJ7tkXSPFX2xIlqZR/iJMw6dkHenq19YL2OIp2/SzKnmejtsWB/RNz9BblsWOw=)

**Comment by chrisnruud** [comment]
Came here to report this as well.

**Comment by lukycody** [comment]
Hey there, we encountered a similiar problem. As our issue was closed recently in favor of this one, let me just add:

As far as I could tell, the issue (not found/matching aria-describedBy element) is unrelated to the "hide-details" prop, that was described as reproduction step here. It occurs without it aswell. So you might want to consider it and have a look at #17730 for a detailed description with code and screenshots. Cheers.

**Comment by lukycody** [comment]
@K-Schaeffer 
Did some further testing with your feedback and encountered, that in our case the dirty prop was problematic and caused the aria-described problem. Removing it also removed the error.
(The "dirty" appearance is the default for the v-selects (label is fixed on top), so we used dirty on the v-text-fields to get the same look - and would like to keep the same appearance)

However, an exception comes with text-fields with type="date" for the datepicker. There it exists even without the dirty prob.
Could you try this on your end again?

**Comment by johnleider** [comment]
I've done some digging on this and it stems from our generation of a **messagesId** that's passed through a scoped slot. This needs to check if messages are available to show. It's a bit involved unfortunately, but it's on my radar.

**Comment by hermes85pl** [comment]
Additionally, there still is an empty `label` element if you don't specify the `label` property, hence if you want to put an external label by yourself (along with other _details_), you end up with two `label` elements for the same field (i.e. your custom one, and the hidden empty label rendered by Vuetify), which affects accessibility.

@johnleider, please also take this into consideration when tackling this issue.

**Comment by sky0matic** [comment]
Accessibility is a main concern for us, let's hope this gets the attention it deserves

**Comment by johnleider** [comment]
> Accessibility is a main concern for us, let's hope this gets the attention it deserves

Looking for help on this one now.
